src/dashboard/components/text_input.py

```diff
@@ -0,0 +1,27 @@
+"""A pre-styled text input component."""
+
+from dash import dcc, html
+
+
+def text_input(id: str, title: str, description: str) -> html.Div:
+    """Pre-styled text input field.
+
+    Args:
+        id (str): input id
+        title (str): input title
+        description (str): placeholder of the input
+
+    Returns:
+        html.Div: A pre-styled input with a title and placeholder
+    """
+    return html.Div(
+        className="flex flex-col",
+        children=[
+            html.P(title, className=""),
```

Comment:
Using a label element would be preferable here. Also applies to `multiline_input.py`.